lib/utils.pm

```diff
@@ -220,13 +220,13 @@ sub is_casp {
     return 1 unless $flavor;
 
     if ($flavor eq 'DVD') {
-        return get_var('FLAVOR') =~ /DVD/;    # Catch DVD and DVD-Staging
+        return get_var('FLAVOR') =~ /DVD/;    # DVD and Staging-?-DVD
     }
     elsif ($flavor eq 'VMX') {
-        return 1;                             # Everything other than DVD is VMX image
+        return get_var('FLAVOR') !~ /DVD/;    # If not DVD it's VMX
```

Comment:
is_casp should tell you if distribution is CaaSP

Optional arguments are:
 - DVD - which returns true if it's ISO image
 - VMX - which returns true if it's virtual image (kvm,vmware,xen,openstack) - everything other than ISO
 - other - which returns true if FLAVOR is matching argument (for is_casp "vmware"...)